client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/components/NonAutogradedProgrammingActionButtonsRow.tsx

```diff
@@ -30,6 +32,8 @@ const NonAutogradedProgrammingActionButtonsRow: FC<Props> = (props) => {
   const question = questions[questionId];
   const { viewHistory } = question;
 
+  const { isCodaveriEnabled } = assessment;
```

Comment:
in my opinion, this should be `isCourseCodaveriEnabled` to make it clear that this is the course attribute, but since this comes from a very old commit, we can leave this for now, since there are already many changes in this PR.